optaplanner-persistence/optaplanner-persistence-jpa/pom.xml

```diff
@@ -171,6 +165,18 @@
       </testResource>
     </testResources>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
For some reason, the dependency plugin does not recognize the `jakarta.transaction-api` as a replacement of the `jboss-transaction-api_1.2_spec`.